tensorflow/core/profiler/internal/gpu/device_tracer_rocm.cc

```diff
@@ -0,0 +1,834 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
```suggestion
/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Reply:
this will be fixed in the next commit/push

Reply:
Hi, I am aware of the intention to pull this into TF 2.5. The quickest way forward is to apply the suggested review comments. Most of these are straightforward refactors. For example, the wrong copyright year might seem inconsequential but we do try to do our part to not raise tech debt. Please unresolve the earlier review feedback that have not been addressed.